window/src/os/windows/window.rs

```diff
@@ -779,12 +781,30 @@ unsafe fn mouse_move(hwnd: HWND, _msg: UINT, wparam: WPARAM, lparam: LPARAM) ->
     }
 }
 
+lazy_static! {
+    static ref WHEEL_SCROLL_LINES: i16 = read_scroll_speed("WheelScrollLines").unwrap_or(3);
+    static ref WHEEL_SCROLL_CHARS: i16 = read_scroll_speed("WheelScrollLines").unwrap_or(3);
```

Comment:
This should be `WheelScrollChars` if you can fix in flight. Otherwise I can update the PR.

Reply:
I've fixed this locally; will push shortly!